src/plugins/kibana_react/public/code_editor/editor_theme.ts

```diff
@@ -13,7 +26,13 @@ import chrome from 'ui/chrome';
 // NOTE: For talk around where this theme information will ultimately live,
 // please see this discuss issue: https://github.com/elastic/kibana/issues/43814
 
-const IS_DARK_THEME = chrome.getUiSettingsClient().get('theme:darkMode');
+// Disable dark theme if unable to query client UI settings
+let IS_DARK_THEME = false;
+try {
+  IS_DARK_THEME = chrome.getUiSettingsClient().get('theme:darkMode');
```

Comment:
You shouldn't be using code from `ui/public` in NP plugins.
This component should receive `CoreStart['uiSettings']` as an argument.
Alternative, if your component is rendered from withing a `KibanaReactContext` (and it should), you may use the `useKibana` hook to fetch `uiSettings`.